ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -15,29 +15,16 @@
       @proceed="onStaffPaymentChanges($event)"
     />
     <v-row no-gutters class="pt-2">
-      <v-col :class="[$style['search-info'], 'select-search-text', 'pt-4']">
-        <span v-html="typeOfSearch">
-        </span>
-        <div v-if="shouldShowFeeHint">
-          <span>
-            Each search incurs a
-          </span>
-          <v-tooltip
-            class="pa-2 pt-2"
-            content-class="top-tooltip"
-            top
-            transition="fade-transition"
-          >
-            <template v-slot:activator="{ on, attrs }">
-              <span v-bind="attrs" v-on="on" :class="$style['fee-info']"> fee of ${{ fee }}.</span>
-            </template>
-            <v-row no-gutters class="pt-2 pb-2">
-              <span>
-                Each search will incur a fee of ${{ fee }},
-                including searches that return no results.
-              </span>
-            </v-row>
-          </v-tooltip>
+      <v-col :class="[$style['search-info'], 'select-search-text']">
+        <div v-if="isStaffBcolReg || isStaffSbc">
+          Select a search category and then enter a value to search. Search fees are as follows:
+          Personal Property search - $10.00; Manufactured Home Search - $10.00;
+          combined Personal Property and Manufactured Home search - $15.00.
+        </div>
+        <div v-else>
+          Select a search category and then enter a value to search. Search fees are as follows:
+          Personal Property search - $8.50; Manufactured Home search - $7.00;
+          combined Personal Property and Manufactured Home search - $12.00.
         </div>
```

Comment:
Looks good but since the values are the only dynamic properties here, could we template those as computed properties vs the whole block?

Reply:
Yea, I was thinking about it, and other than saving a few lines of code I didn't see any other benefit of templating. But sure, let me refactor a bit.

Reply:
Updated.